src/wp-content/themes/twentythirteen/inc/back-compat.php

```diff
@@ -35,7 +35,7 @@ function twentythirteen_switch_theme() {
  */
 function twentythirteen_upgrade_notice() {
 	printf(
-		'<div class="error"><p>%s</p></div>',
+		'<div class="notice notice-error"><p>%s</p></div>',
```

Comment:
Don't change back-compat classes, as the context where these run may not include the necessary support.